src/App.js

```diff
@@ -1,14 +1,31 @@
-import React from 'react';
-import Calculator from './Components/Calculator';
+import React from "react";
+import Calculator from "./Components/Calculator";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import Home from "./pages/Home";
+import Quote from "./pages/Quote";
+import ErrorPage from "./pages/ErrorPage";
+import NavBar from "./Components/NavBar";
+import CalcPage from "./pages/CalcPage";
 
-class App extends React.Component{
-  render() { 
+class App extends React.Component {
+  render() {
     return (
-      <div className="App">
-        <div className='wholePage'>
-          <Calculator />
-        </div>
-      </div>
+      <Router>
+        <NavBar />
+
+        <Routes>
+          <Route path="/" element={ <Home /> } />
+          <Route path="/calc" element = {<CalcPage/>} />
+          <Route path="/quote" element = {<Quote />} />
+          <Route path="*" element = {<ErrorPage />}/>
+        </Routes>
+      </Router>
+
+      // <div className="App">
+      //   <div className='wholePage'>
+      //     <Calculator />
+      //   </div>
+      // </div>
```

Comment:
- [ ] Please kindly remove commented code from your code base, as this makes your code base dirty and less readable.